ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-and-3node-validator-ranGen.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  # The name will be used to generate the placementBinding and placementRule names as {name}-placementBinding and {name}-placementRule
+  name: "group-du-master-validator"
+  namespace: "ztp-group"
+spec:
+  bindingExcludedRules:
+    # This policy will correspond to all clusters without this label:
```

Comment:
This label "ztp-done" will be added by TALCO ? if yes, please add this info to the description and tell end-user must not  change that otherwise TALCO will miss behave.